packages/analytics/src/helpers.ts

```diff
@@ -40,6 +55,30 @@ export function promiseAllSettled<T>(
   return Promise.all(promises.map(promise => promise.catch(e => e)));
 }
 
+/**
+ * Creates a TrustedTypePolicy object that implements the rules passed as policyOptions.
+ *
+ * @param policyName A string containing the name of the policy
+ * @param policyOptions Object containing implementations of instance methods for TrustedTypesPolicy, see {@link https://developer.mozilla.org/en-US/docs/Web/API/TrustedTypePolicy#instance_methods
+ * | the TrustedTypePolicy reference documentation}.
+ * @returns
```

Comment:
Probably delete this if you don't want to add a text description. It will pick up the type from Typescript, you only need this if you want to write some additional text to describe the return value.